-------------------------------------------------------------------------------
Test set: TestSuite
-------------------------------------------------------------------------------
Tests run: 3, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 121.891 s <<< FAILURE! - in TestSuite
TestCase3(Tests.Tests)  Time elapsed: 0.42 s  <<< FAILURE!
java.lang.AssertionError: expected [2] but found [0]
	at Tests.Tests.TestCase3(Tests.java:50)

